### DISCUSSION TITLE
Recover deleted repository issue(s)

### DISCUSSION THREAD

**Discussion by anthonyylee** [community/community#47797] [Repositories] [answered]
### Select Topic Area

Question

### Body

I deleted an issue on one of my organization's private repo and would like to recover to close it instead so to maintain a consistent record. However, I can't find a way to recover the deleted issue.

Is there a way that Github's staff would be able to help recover the deleted issue?

Thanks,

**Comment by radar-blip** [comment, marked as answer]
You could try here: [Support Page](https://support.github.com/request) under `Restore deleted data`

**Reply by anthonyylee** [reply]
Thank you! There is indeed a specific button for requesting support for deleted data.

**Reply by radar-blip** [reply]
You're welcome. Hope it worked out.
